### PR TITLE
flake8 and pylint fixes

### PR DESCRIPTION
**Pull request by bje-**
Self-evident.

### CODE REVIEW

File: examples/scigrid-de/add_load_gen_trafos_to_scigrid.py [hunk 1]
```diff
@@ -92,7 +92,7 @@
 # note that some columns have 'quotes because of fields containing commas'
 vertices = pd.read_csv(
     folder_prefix + "scigrid-151109/vertices_de_power_151109.csvdata",
-    sep=",",
+    sep=", ",
```
Comment (FabianHofmann) [inline examples/scigrid-de/add_load_gen_trafos_to_scigrid.py:95]: This is a separator and csv reading does not like the the space after the comma, could you double-check whether this works as expected?

File: examples/scigrid-de/add_load_gen_trafos_to_scigrid.py [hunk 2]
```diff
@@ -103,7 +103,7 @@
 
 links = pd.read_csv(
     folder_prefix + "scigrid-151109/links_de_power_151109.csvdata",
-    sep=",",
+    sep=", ",
```
Comment (FabianHofmann) [inline examples/scigrid-de/add_load_gen_trafos_to_scigrid.py:106]: Same here

File: examples/scigrid-de/add_load_gen_trafos_to_scigrid.py [hunk 3]
```diff
@@ -534,7 +534,7 @@ def read_kraftwerksliste(with_latlon=True):
         delimiter=";",
         encoding="utf-8",
         thousands=".",
-        decimal=",",
+        decimal=", ",
```
Comment (FabianHofmann) [inline examples/scigrid-de/add_load_gen_trafos_to_scigrid.py:537]: same here

**Review by FabianHofmann — COMMENTED** [review]
Looks mostly good, see the comment below

File: examples/scigrid-de/add_load_gen_trafos_to_scigrid.py (hunk 1 above)
Comment (fneum) [inline examples/scigrid-de/add_load_gen_trafos_to_scigrid.py:95]:
```suggestion
    sep=",",
```

File: examples/scigrid-de/add_load_gen_trafos_to_scigrid.py (hunk 2 above)
Comment (fneum) [inline examples/scigrid-de/add_load_gen_trafos_to_scigrid.py:106]:
```suggestion
    sep=",",
```

File: examples/scigrid-de/add_load_gen_trafos_to_scigrid.py (hunk 3 above)
Comment (fneum) [inline examples/scigrid-de/add_load_gen_trafos_to_scigrid.py:537]:
```suggestion
        decimal=",",
```

Comment (fneum) [inline examples/scigrid-de/add_load_gen_trafos_to_scigrid.py:537]:
```suggestion
        decimal=",",
```

File: pypsa/linopf.py [hunk 4]
```diff
@@ -863,7 +863,7 @@ def get_period(n, glc, sns):
     )
     substr = lambda s: re.sub(r"[\[\]\(\)]", "", s)
     for name, glc in glcs.iterrows():
-        car = [substr(c.strip()) for c in glc.carrier_attribute.split(",")]
+        car = [substr(c.strip()) for c in glc.carrier_attribute.split(", ")]
```
Comment (fneum) [inline pypsa/linopf.py:866]:
```suggestion
        car = [substr(c.strip()) for c in glc.carrier_attribute.split(",")]
```

File: pypsa/linopf.py [hunk 5]
```diff
@@ -896,7 +896,7 @@ def get_period(n, glc, sns):
     # (3) transmission_expansion_cost_limit
     glcs = n.global_constraints.query("type == " '"transmission_expansion_cost_limit"')
     for name, glc in glcs.iterrows():
-        car = [substr(c.strip()) for c in glc.carrier_attribute.split(",")]
+        car = [substr(c.strip()) for c in glc.carrier_attribute.split(", ")]
```
Comment (fneum) [inline pypsa/linopf.py:899]:
```suggestion
        car = [substr(c.strip()) for c in glc.carrier_attribute.split(",")]
```

**Review by fneum — CHANGES_REQUESTED** [review]

**Review by fneum — APPROVED** [review]

File: examples/scigrid-de/add_load_gen_trafos_to_scigrid.py (hunk 1 above)
Comment (bje-) [reply examples/scigrid-de/add_load_gen_trafos_to_scigrid.py]: In the version of Pandas I have, whitespace is cleaned up as you would expect. `sep=','` is sufficient.

File: examples/scigrid-de/add_load_gen_trafos_to_scigrid.py (hunk 2 above)
Comment (bje-) [reply examples/scigrid-de/add_load_gen_trafos_to_scigrid.py]: Ditto.

**Pull request merged**